source/MaterialXView/Editor.h

```diff
@@ -37,10 +38,14 @@ class PropertyEditor
     void create(Viewer& parent);
     void addItemToForm(const mx::UIPropertyItem& item, const std::string& group,
                        ng::Widget* container, Viewer* viewer, bool editable);
-      
+    ng::FloatBox<float>* makeFloatWidget(ng::Widget* container, const std::string label, mx::ValuePtr value,
+                       bool editable, mx::ValuePtr min, mx::ValuePtr max, Viewer* viewer, const std::string& path);
+
     bool _visible;
     ng::Widget* _container;
     ng::Window* _formWindow;
+    ng::GridLayout* _gridLayout2;
+    ng::GridLayout* _gridLayout3;
```

Comment:
Do you need a destructor to handle lifetime of these pointers?

Reply:
No. These are just pointers for easy access. They are owned by the parent widget when you add them as children.
That's why most widgets you just new them and add them as children.